### ISSUE TITLE
[iceprod2] create self-signed cert on website startup (Trac #1483)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1475]
If nginx is found and a cert isn't provided to us, create a self-signed cert to use and add it to the config.

Notes:
- need config option to track whether this is auto-generated
- if config cert is autogenerated and can't be found, make a new one
- if config cert is manual and can't be found, raise exception and die

Migrated from https://code.icecube.wisc.edu/ticket/1483
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "If nginx is found and a cert isn't provided to us, create a self-signed cert to use and add it to the config.\n\nNotes:\n- need config option to track whether this is auto-generated\n- if config cert is autogenerated and can't be found, make a new one\n- if config cert is manual and can't be found, raise exception and die\n", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] create self-signed cert on website startup", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-12-11T19:28:26", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151211T23:34:01: 
* david.schultz commented:

> In changeset 140350:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140350"
> handle ssl certs in website/nginx better. fix some old things. better logging. closes #1475
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
